Library/Source/RuntimeUWP.cpp

```diff
@@ -15,6 +15,7 @@ namespace Babylon
         : Runtime{std::make_unique<RuntimeImpl>(window, rootUrl)}
         , m_window{window}
     {
+        NativeEngine::InitializeDeviceContext(window, 32, 32);
```

Comment:
I know this is what it was doing before, but ideally, we really should figure out the size of the native window here and possibly all the platforms, like what Win32 one does.